Loop/Managers/DeviceDataManager.swift

```diff
@@ -734,7 +734,6 @@ extension DeviceDataManager {
 
     func didBecomeActive() {
         updatePumpManagerBLEHeartbeatPreference()
-        refreshDeviceData()
```

Comment:
Hmm, this seems like the wrong fix. It seems to me that `refreshDeviceData` should be able to be called at any time and you should end up with the same result. A recommendation based upon timing seems like it would present problems.

What do you think @ps2?